rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -643,7 +647,12 @@ def generate_wrapping_rosdoc2_sphinx_project_into_directory(
             'user_conf_py_filename': esc_backslash(
                 os.path.abspath(os.path.join(user_sourcedir, 'conf.py'))),
             'breathe_projects': ',\n'.join(breathe_projects) + '\n    ',
-            'intersphinx_mapping_extensions': ',\n        '.join(intersphinx_mapping_extensions)
+            'intersphinx_mapping_extensions': ',\n        '.join(intersphinx_mapping_extensions),
+            'package': package,
+            'package_authors': ', '.join(set(
```

Comment:
```suggestion
            'package_authors': ', '.join(sorted(set(
```

One tweak to sort the output for repeatability.